packages/react-components/src/components/MentionPopover.tsx

```diff
@@ -314,18 +315,20 @@ export const _MentionPopover = (props: _MentionPopoverProps): JSX.Element => {
         <Stack.Item styles={headerStyleThemed(theme)} aria-label={title}>
           {getHeaderTitle()}
         </Stack.Item>
-        <Stack
-          /* @conditional-compile-remove(mention) */
-          data-ui-id={ids.mentionSuggestionList}
-          className={suggestionListStyle}
-        >
-          {suggestions.map((suggestion, index) => {
-            const active = index === activeSuggestionIndex;
-            return onRenderSuggestionItem
-              ? onRenderSuggestionItem(suggestion, onSuggestionSelected, active)
-              : defaultOnRenderSuggestionItem(suggestion, onSuggestionSelected, active);
-          })}
-        </Stack>
+        <FocusZone shouldFocusOnMount={true}>
```

Comment:
When MentionPopover is presented, the focused element should still be the inputbox.
The first element of the popover should be "active", not focused.

Reply:
So without the focus zone, it won't read out the first item in the list, similar to the participant pane. You're right though, it does affect the focus of the cursor. 

I'll look into the active param you mentioned.